build-all.sh

```diff
@@ -718,6 +738,10 @@ then
 fi
 export SED
 export RELEASE_NAME
+if [ "x${TLS_SUPPORT}" != "x" ]
```

Comment:
Is there a reason not to export empty TLS_SUPPORT? build-uclibc.sh compares with "yes" anyway so empty TLS_SUPPORT would behave like TLS_SUPPORT=no. In fact, in current configuration TLS_SUPPORT always is either yes or no, so this condition is always true.